assets/src/edit-story/components/checklistCard/styles.js

```diff
@@ -33,6 +34,16 @@ export const Wrapper = styled.div`
   background: ${({ theme }) => theme.colors.bg.secondary};
 `;
 
+export const FooterText = styled(Text).attrs({
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL,
+  as: 'div',
```

Comment:
is this why the `div` Text typography setting was added? 

if so, I think a better approach might be to create a `DefaultList` for the checklistCard with the color setting there instead of through this div? Could even through it in the `CardListWrapper` instead, which would be pretty much the same approach as this but respecting text elements. Thoughts?

Reply:
I just added it because I was getting complaints about putting a `ul` in a `p` tag. Not sure if we'd run into that issue with a `span` tho 🤔 

Can try it out with a span and remove the `div` option if it doesn't complain

Reply:
If you look at what I did with some of the other checks where the copy is a fragment with list items, i am using a `<List>` tag and a wrapper that I added in the checklistCard last week to style it. That should get you out of this necessity.